hathor/websocket/factory.py

```diff
@@ -120,7 +146,7 @@ def _schedule_and_send_metric(self):
             'type': 'dashboard:metrics',
             'time': reactor.seconds(),
         }
-        self.broadcast_message(data)
+        self.send_or_enqueue(data)
```

Comment:
Is it compatible with the current clients of the websocket? I think we can keep compatibility adding a new message `clear-subscriptions` that will be called by new clients. In the future, after every client has been updated, we can start with no subscriptions from the beginning.

Reply:
Yes, it's fully compatible. The method `send_or_enqueue` will call `broadcast_message` for this message type, however it will also handle sending this message to the subscribed connections.

Reply:
This behavior is already implemented as the websocket is removed from the broadcast group if it subscribes to any message type.

Reply:
Yes, I forgot to comment about the `clear-subscriptions`. Every time you subscribe to messages I remove the connection to the broadcast set.